README.md

```diff
@@ -14,18 +14,15 @@ you to choose wisely and put in the work once you get to high school.
 
 [![View Schools](app/assets/images/view_schools_button_readme.png)](http://lucky8.herokuapp.com/schools)
 
-
-Ruby on Rails
+Getting Started for Teachers
 -------------
+This app help you and your students look up high school informations in Boston area.
+To help your students understand how this app works, we created a scavenger hunt as lesson plan for you.
+You can print out the sheet below, and ask students to fill out the numbers in 10-15 mins.
+https://github.com/geoffreylitt/lucky8/raw/master/ScavengerHunt_HSGuideSite.docx
```

Comment:
This could be a link with text rather than just the URL. Syntax is [link title](url)